src/components/PublicProjectCard.tsx

```diff
@@ -31,25 +32,34 @@ export function PublicProjectCard({ project, activityPromise }: Props) {
   return (
     <Card className="w-full max-w-fit" style={{ order }}>
       <CardHeader>
-        <CardTitle className="flex items-center gap-2">
-          <Image
-            width={40}
-            height={40}
-            src={`https://github.com/${project.ownerLogin}.png?size=80`}
-            alt={project.ownerLogin}
-            className={`size-10 bg-gray-300 ${
-              project.ownerType === "User" ? "rounded-full" : "rounded-md"
-            }`}
-          />
-          <a
-            href={`https://github.com/${project.ownerLogin}/${project.name}`}
-            target="_blank"
-            className="text-nowrap"
-          >
-            <span>{project.ownerLogin}/</span>
-            <wbr />
-            <span>{project.name}</span>
-          </a>
+        <CardTitle className="flex items-center justify-between">
+          <div className="flex items-center gap-2">
+            <Image
+              width={40}
+              height={40}
+              src={`https://github.com/${project.ownerLogin}.png?size=80`}
+              alt={project.ownerLogin}
+              className={`size-10 bg-gray-300 ${
+                project.ownerType === "User" ? "rounded-full" : "rounded-md"
+              }`}
+            />
+            <a
+              href={`https://github.com/${project.ownerLogin}/${project.name}`}
+              target="_blank"
+              className="text-nowrap"
+            >
+              <span>{project.ownerLogin}/</span>
+              <wbr />
+              <span>{project.name}</span>
+            </a>
+          </div>
+
+          {lastCommit && (
+            <span className="flex items-center gap-1 text-xs font-medium">
+              <History className="h-4 w-4" /> Last commit:{" "}
+              {new Date(lastCommit).toLocaleDateString("hi-IN")}
```

Comment:
```suggestion
              {new Date(lastCommit).toLocaleDateString()}
```

I think we haven't defined a standard yet, probably let's keep it without explicit locale for now.